appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/DesignToolbar.ui.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2009-2011 Google, All Rights reserved -->
+<!-- Copyright 2011-2023 MIT, All rights reserved -->
+<!-- Released under the Apache License, Version 2.0 -->
+<!-- http://www.apache.org/licenses/LICENSE-2.0 -->
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+             xmlns:g="urn:import:com.google.gwt.user.client.ui"
+             xmlns:ai="urn:import:com.google.appinventor.client.widgets"
+             xmlns:ya="urn:import:com.google.appinventor.client.editor.youngandroid.actions"
+             ui:generatedFormat="com.google.gwt.i18n.server.PropertyCatalogFactory"
+             ui:generatedKeys="com.google.gwt.i18n.server.keygen.MethodNameKeyGenerator"
+             ui:generateLocales="default">
+  <ui:with field="messages" type="com.google.appinventor.client.OdeMessages" />
+
+  <ai:Toolbar>
+    <g:Label ui:field="projectNameLabel" styleName="ya-ProjectName" />
+
+    <!-- Left button items -->
+    <ai:ToolbarItem name="TutorialToggle" caption="{messages.toggleTutorialButton}"
+                    visible="false" align="center">
+      <ya:ToggleTutorialAction />
+    </ai:ToolbarItem>
+    <ai:DropDownButton name="ScreensDropdown" caption="{messages.screensButton}"
+                       ui:field="pickFormItem" align="center" />
+    <ai:ToolbarItem name="AddForm" caption="{messages.addFormButton}" ui:field="addFormItem"
+                    align="center">
+      <ya:AddFormAction />
+    </ai:ToolbarItem>
+    <ai:ToolbarItem name="RemoveForm" caption="{messages.removeFormButton}" ui:field="removeFormItem"
```

Comment:
Note that on Screen1 this should look disabled. The behavior seems correct but the CSS or something is not.

Reply:
Hmm. This only seems to be the case when in read only mode.